index.js

```diff
@@ -4873,12 +4894,16 @@ export async function createCsharpBom(path, options) {
       if (dlist?.length) {
         pkgList = pkgList.concat(dlist);
       }
+      if(DEBUG_MODE){
+        console.log(`pkgList List Size :: ${pkgList.length}`);
+      }
     }
+    
     if (pkgList.length) {
-      console.log(
-        `Found ${pkgList.length} components by parsing the ${csProjFiles.length} csproj files. The resulting SBOM will be incomplete.`,
-      );
-      options.failOnError && process.exit(1);
+     // console.log(
```

Comment:
Any ideas why this is commented out? This message is useful to indicate the absence of lock files for all users. Perhaps, we can suppress this only for .Net framework somehow.

Reply:
Yeah we should find a way for saying the incomplete sbom. I dont think its best idea based on this.